RecoJets/JetProducers/BuildFile.xml

```diff
@@ -10,6 +10,7 @@
 <use name="SimDataFormats/Vertex"/>
 <use name="SimDataFormats/Track"/>
 <use name="RecoJets/JetAlgorithms"/>
+<use name="CondFormats/EcalObjects"/>
```

Comment:
I guess this should go to ` RecoJets/JetProducers/plugins/BuildFile.xml` instead.